www/docs/mp/index.php

```diff
@@ -494,6 +494,17 @@
 
         break;
 
+    case 'recent':
+        $policiesList = new MySociety\TheyWorkForYou\Policies;
+        $positions = new MySociety\TheyWorkForYou\PolicyPositions( $policiesList, $MEMBER );
+        $divisions = new MySociety\TheyWorkForYou\Divisions($MEMBER, $positions, $policiesList);
+
+        $data['divisions'] = $divisions->getRecentMemberDivisions();
+        //print_r($data['divisions']);
+        //exit();
+
+        MySociety\TheyWorkForYou\Renderer::output('mp/recent', $data);
+      break;
```

Comment:
The indentation here isn't consistent with the other case statements here (although I don't know what standard we follow for PHP - there's none mentioned here: http://mysociety.github.io/coding-standards.html )